treeherder/etl/seta.py

```diff
@@ -78,3 +85,56 @@ def job_priorities_to_jobtypes():
         jobtypes.append(jp.unique_identifier())
 
     return jobtypes
+
+
+def get_reference_data_names(project, build_system):
```

Comment:
This is simply moving the function from `treeherder/seta/job_priorities.py` into this module.